src/containers/NdlaFilm/components/NdlaFilmArticle.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree. *
+ */
+
+import { useField, useFormikContext } from 'formik';
+import { useCallback, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import styled from '@emotion/styled';
+import { IconButtonV2 } from '@ndla/button';
+import { colors, spacing } from '@ndla/core';
+import { DeleteForever } from '@ndla/icons/editor';
+import Tooltip from '@ndla/tooltip';
+import { IArticleSummaryV2 } from '@ndla/types-backend/article-api';
+import { IMultiSearchSummary } from '@ndla/types-backend/search-api';
+import DropdownSearch from '../../NdlaFilm/components/DropdownSearch';
+import { getUrnFromId, getIdFromUrn } from '../../../util/ndlaFilmHelpers';
+import { searchArticles } from '../../../modules/article/articleApi';
+
+interface Props {
+  fieldName: string;
+  onUpdateArticle: Function;
+}
+
+const ArticleElement = styled.div`
+  align-items: center;
+  background: ${colors.brand.greyLighter};
+  display: flex;
+  justify-content: space-between;
+  margin: 0 0 ${spacing.normal};
+  padding: ${spacing.small};
+`;
+
+const NdlaFilmArticle = ({ fieldName, onUpdateArticle }: Props) => {
+  const { t } = useTranslation();
+  const form = useFormikContext();
+  const [field] = useField<string>(fieldName);
+  const [selectedArticle, setSelectedArticle] = useState<null | IArticleSummaryV2>(null);
+
+  const fetchArticle = useCallback(async () => {
+    if (field.value) {
+      const article = await searchArticles({ ids: `${getIdFromUrn(field.value)}` });
+      setSelectedArticle(article.results[0]);
+    } else {
+      setSelectedArticle(null);
+    }
+  }, [field.value]);
+
+  useEffect(() => {
+    fetchArticle();
+  }, [field.value, fetchArticle]);
+
+  return (
+    <>
+      {selectedArticle && (
+        <ArticleElement>
+          <Link
+            to={`/subject-matter/learning-resource/${selectedArticle.id}/edit/${selectedArticle.title.language}`}
```

Comment:
Trur det finnes en funksjon som gjør dette for deg. Så slipper vi å ha pather mange plasser.